common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -6,6 +6,7 @@
   "core.wynntils.category.redirects": "Game Notification Redirects",
   "core.wynntils.category.tooltips": "Item Tooltips",
   "core.wynntils.category.uncategorized": "Uncategorized",
+  "feature.wynntils.BlacksmithRedirectFeature.name": "Blacksmith Message Redirect",
```

This should not even run in dev env.
```suggestion
  "feature.wynntils.blacksmithRedirect.name": "Blacksmith Message Redirect",
```